SPTDataLoaderTests/SPTDataLoaderFactoryTest.m

```diff
@@ -165,21 +165,27 @@ - (void)testOfflineChangesCachePolicy
 - (void)testRelayToDelegateWhenPerformingRequest
 {
     SPTDataLoaderRequest *request = [SPTDataLoaderRequest new];
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wnonnull"
     [self.factory requestResponseHandler:nil performRequest:request];
+#pragma clang diagnostic pop
```

Comment:
I like it too. In this case though I kind of want the warning silencers to be an eyesore, because we should ideally fix the issue and remove them. I just added them now since it was the smaller change, and I wanted to increase my confidence in the commit my not altering the tests too much. Feel free to change this to whatever code style you prefer though, for me it doesn't matter.